syftlib/src/main/java/org/openmined/syft/execution/Plan.kt

```diff
@@ -35,12 +37,17 @@ class Plan(val planId: String) {
         val x = trainingBatch.first
         val y = trainingBatch.second
 
+        // batchSize is the pytorch IValue tensor containing the batchSize specified in the client configs.
         val batchSize = IValue.from(
             Tensor.fromBlob(longArrayOf(clientConfig.batchSize), longArrayOf(1))
         )
+
+        // lr is the pytorch IValue tensor containing the learning rate specificed in the client configs.
         val lr = IValue.from(
             Tensor.fromBlob(floatArrayOf(clientConfig.lr), longArrayOf(1))
         )
+
+        // We feed in the training data, output labels, batchSize, learning rate and other parameters to the forward function of the pytorch module.
```

Comment:
May be break the comment into two lines. Also mention the output of the forward.